packages/common/src/stores/utils.ts

```diff
@@ -1,5 +1,5 @@
 import type { State, StateCreator } from 'zustand';
-import create from 'zustand';
+import { create } from 'zustand';
```

Comment:
The `State` type is deprecated, v5 is coming soon so let's remove it.

```diff
- export const createStore = <T extends State>
+ export const createStore = <T>

Reply:
True, thanks